0-bootstrap/modules/jenkins/README.md

```diff
@@ -0,0 +1,134 @@
+## Overview
+
+The objective of this module is to deploy a Google Cloud Platform project `prj-cicd` to host a Jenkins Agent that can be used to deploy your infrastructure changes. This module is a replica of the [cloudbuild module](https://github.com/terraform-google-modules/terraform-google-bootstrap/tree/master/modules/cloudbuild), but re-purposed to use jenkins instead of Cloud Build. This module creates:
+- The `prj-cicd` project to hold the Jenkins Agent
+- GCE Instance for the Jenkins Agent, assigning SSH public keys in the metadata to allow connectivity from the Jenkins Master.
+- FW rules to allow communication over port 22
+  - TODO: use a fixed IP or no public IP at all
```

Comment:
IMO no public IP at all makes most sense. Is there guidance in the whitepaper?

Reply:
I need the public IP to be able to test connectivity, even if it is Ephemeral. But the withe paper says there will be VPN with on prem. I can also remove the public IP once I can deploy the Jenkins Master in the same VPC for testing purposes.